src/app/public/modules/modal/modal-adapter.service.ts

```diff
@@ -11,24 +12,39 @@ export class SkyModalAdapterService {
 
   private docRef: any;
   private bodyEl: HTMLElement;
+  private renderer: Renderer2;
+  private hostRef: ComponentRef<SkyModalHostComponent>;
 
   constructor(
-    private windowRef: SkyWindowRefService) {
+    private rendererFactory: RendererFactory2,
+    private windowRef: SkyWindowRefService
```

Comment:
Adding dependencies is a breaking change because a consumer could be mocking out the service in their unit tests. Any new dependencies should be added at the end of the list, and made optional.

See: https://docs.blackbaud.com/skyux-team-docs/style#constructors

Reply:
Done